src/main/java/com/airlines/model/CrewMember.java

```diff
@@ -0,0 +1,71 @@
+package com.airlines.model;
+
+import java.time.LocalDate;
+
+public class CrewMember {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String position;
```

Comment:
position should be enum, justification a little below

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/1269dee2e4effc40beaddfca6d85b7799518548f